hack/config_crd.go

```diff
@@ -0,0 +1,19 @@
+// +kubebuilder:object:generate=true
+// +groupName=config.microshift.openshift.io
+// +versionName=v1
+
+// This file only exists to help translate our config struct
+// to an OpenAPIV3 spec via controller-gen, it should not be exported
```

Comment:
What brings controller-gen into the build environment?

Reply:
Currently, that would be just doing a simple `go install` similar to the [existing lichen](https://github.com/openshift/microshift/blob/main/scripts/fetch_tools.sh#L153), I'll add that in this PR

Reply:
Added controller-gen to the fetch script, and it should be pulled in during the config-verify now